src/Model/Order/Order.php

```diff
@@ -103,12 +152,12 @@ public function toNative(): array
     }
 
     public static function create(
-        OrderId $id,
-        BusinessName $businessName,
-        BusinessAddress $businessAddress,
-        ContactPerson $contactPerson,
-        OrderIsPaid $isPaid,
-        OrderAmount $amount
+        NonNullOrderId $id,
+        NonNullBusinessName $businessName,
+        NonNullBusinessAddress $businessAddress,
+        NonNullContactPerson $contactPerson,
+        NonNullPaymentStatus $paymentStatus,
```

Comment:
Since the payment status defaults to UNPAID when the `OrderWasCreated`event is applied it seems to me that the $paymentStatus argument is unnecessary.

Reply:
Domain question: Can an order be created with a different payment status than UNPAID?

Reply:
No